src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegalinstantiation/Example1.java

```diff
@@ -0,0 +1,34 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="IllegalInstantiation">
+      <property name="classes"
```

Comment:
Example1 must be default config.
All other configs are showing change in behavior after customization by properties

Reply:
In my previous point I meant to copy example3 Java statements only.